### PR TITLE
don't run azure on other branches either

### PR DESCRIPTION
**Pull request by metagn**
refs #20184

Azure is more prone to clogging & it's especially bad with the timeouts (some PRs haven't rebased).

### CODE REVIEW

**Comment by ringabout** [comment]
What's the difference? Does it run on other branches before?

**Comment by metagn** [comment]
Yes.

image

**Comment by ringabout** [comment]
Sure, thanks for your explanation!

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 trigger:
   branches:
     include:
-    - '*'
+    - 'devel'
+    - 'version-1-6'
+    - 'version-1-2'
+    - 'version-1-0'
```
Comment (Varriount) [inline azure-pipelines.yml]: Can this be modified to something like
```suggestion
    - 'version-*
```
so that future versions are covered?

Comment (metagn) [reply azure-pipelines.yml]: Should work

File: azure-pipelines.yml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 trigger:
   branches:
     include:
-    - '*'
+    - 'devel'
+    - 'version-*
```
Comment (metagn) [inline azure-pipelines.yml]:
```suggestion
    - 'version-*'
```

**Pull request merged**